SECURITY.md

```diff
@@ -0,0 +1,51 @@
+# Security Policy
+
+## How we do security
+
+As much as possible, DJL Serving relies on automated tools to do security scanning. In particular, we support:
+
+1. Docker CVE patch scanning: Using AWS ECR
+2. Dependency Analysis: Using Dependabot
+3. Code Analysis: Using CodeQL
+
+## Important Security Guidelines
+
+1. DJL Serving has two APIs: inference and management. More can be available through plugins.
+    HTTP - `8080` for both the inference and management API
+
+    By default, both APIs are available on port `8080` through HTTP and accessible to `localhost`.  The addresses can be configured by following the guide for
+    [global configuration](serving/docs/configurations_global.md).
+    DJL Serving does not prevent users from configuring the address to be of any value, including the wildcard address `0.0.0.0`.
+    Please be aware of the security risks of configuring the address to be `0.0.0.0` as this will give all addresses (including publicly accessible addresses)
+    on the host, access to the TorchServe endpoints listening on the ports shown above.
+    You should be especially careful with the management API including setting it to a publicly accessible address or forwarding it's port to one.
+    If expose, it could allow an attacker to create models and execute arbitrary code on your machine.
+2. By [default](serving/docker/Dockerfile), the docker images are configured to expose the port `8080` to the host.
+   The default DJL Serving configuration in the container, which is executed by the docker entrypoint, will expose both the inference and management APIs set to `http://0.0.0.0:8080`.
+   This is designed for internal isolated services or development work. For other use cases, provide alternative configurations to avoid exposing the management API.
+
+3. Be sure to validate the authenticity of all model files and model artifacts being used with DJL Serving.
```

Comment:
We might want to mention about
1. `requirements.txt`
2.  options.trust_remote_code
3. environment variable

Reply:
I added mentions on those topics as well. Feel free to let me know or just change it if it doesn't match up with what you had in mind